conf/i18n/localebuildutils.js

```diff
@@ -21,9 +20,8 @@ function copyAssetsForLocales (assetNames) {
         : `${assetName}`;
       LANGUAGES_TO_LOCALES[language].forEach((locale) => {
         const localeBundleName = `${locale}-${assetName}`;
-        src(`./dist/${languageBundleName}`)
-          .pipe(rename(localeBundleName))
-          .pipe(dest('./dist/'));
+        fs.symlinkSync(`./${languageBundleName}`,
```

Comment:
Out of curiosity, do you have a sense for how much space the `symlink` will save in CircleCI? Does this need to be `./dist/${languageBundleName}`?

Reply:
Just checked, and it saves about 1GB. And no, the languageBundleName path doesn't need to include dist. The first param in `symLinkSync` is relative to the second params's parent directory according to the [docs](https://nodejs.org/api/fs.html#fs_fs_symlink_target_path_type_callback). I also confirmed that all the locale builds are available in S3 from the right directory